ChangeLog

```diff
@@ -38,6 +38,7 @@
 === Miscellaneous ===
 
 === Bug fixes ===
+ * #870 (Problem with TensorizedCovarianceModel with spatial dim > 1)
```

Comment:
could you move this change to the 1.10 section ? I'll likely backport that fix

Reply:
IMO it is too late, we do not know whether kriging works in this case, this can wait for 1.11.

Reply:
ok